tools/build/ms-plugins.coffee

```diff
@@ -0,0 +1,113 @@
+{ walkFiles } = require('@resin.io/doxx-utils')
+HbHelper = require('@resin.io/doxx-handlebars-helper')
+
+path = require('path')
+fse = require('fs-extra')
+semver = require('semver')
+
+utils = require('../lib/utils')
+pythonHelper = require('../lib/python')
+
+
+removeFile = (obj, file, files) ->
+	if obj and obj is utils.UNSUPPORTED
```

Comment:
no need to check if `obj` is true-ish, can simply be: `if obj is utils.UNSUPPORTED`
`obj` is also not the best name here, should be like `isSupported` or something like that.